package.json

```diff
@@ -108,14 +108,15 @@
     "tiny-typed-emitter": "^2.1.0",
     "uint8array-extras": "^0.5.0",
     "utm": "^1.1.1",
+    "valibot": "^0.42.1",
```

Comment:
I'm introducing this dep for response body validation, which is used when interfacing with the map server